addons/website_slides/models/slide_channel.py

```diff
@@ -121,10 +121,10 @@ def _get_default_enroll_msg(self):
         string="Promoted Content", default='latest', required=False,
         help='Depending the promote strategy, a slide will appear on the top of the course\'s page :\n'
              ' * Latest Published : the slide created last.\n'
-             ' * Most Voted : the slide which have to most vote.\n'
+             ' * Most Voted : the slide which have to most votes.\n'
```

Comment:
* which **has the** most votes
or
* the **slides** which have **the** most votes

Reply:
good catch, thanks